internal/cypress/docker/runner.go

```diff
@@ -222,6 +233,12 @@ func (r *Runner) teardown(logDir string) error {
 		}
 	}
 
+	// checks that container exists before stopping and removing it
```

Comment:
That's not where the issue occurs though 😅 If the container is already stopped/removed, calling stop/remove in such a case will not cause an error. It's the step before, where we try to copy something out of the container (https://github.com/saucelabs/saucectl/pull/119/files#diff-7209dd8c5e329be195704479efc81def9109e635ba88d753c9b857baae4c21adL220).

Reply:
We agreed about possible cases and here is a quick summary.  1. When the container was stopped for some reason and we try to stop and remove it one more time. 2. When the user forgot to specify the docker image in the config file.